samples/3d-platformer/3d-platformer.gdnlib

```diff
@@ -0,0 +1,14 @@
+[entry]
+
+X11.64="res://build/bin/linuxX64/debugShared/lib3d_platformer.so"
```

Comment:
IMO we should add the ones for windows and macOs as well right away

Reply:
Ahh, this is because of me only running `buildLinuxX64` locally. I'll add the others.

Reply:
@chippmann This file is generated, and it is impossible to know the path of the shared libs in windows and osx from a linux machine as the paths are only calculated when a link task is executed. Maybe it's best to just ignore this file? You probably will trigger a separate build for each platform you deploy to when releasing, so the correct paths will be present here.

Reply:
Yeah you are right. But i would not ignore it in the .gitignore if thats what you mean.